torchgeo/datasets/spacenet.py

```diff
@@ -271,6 +289,53 @@ def _download(self, collections: List[str], api_key: Optional[str] = None) -> No
             print("Extracting...")
             extract_archive(archive_path)
 
+    def plot(
+        self,
+        sample: Dict[str, Tensor],
+        show_titles: bool = True,
+        suptitle: Optional[str] = None,
+    ) -> Figure:
+        """Plot a sample from the dataset.
+
+        Args:
+            sample: a sample returned by :meth:`__getitem__`
+            show_titles: flag indicating whether to show titles above each panel
+            suptitle: optional string to use as a suptitle
+
+        Returns:
+            a matplotlib Figure with the rendered sample
```

Comment:
Can you add:
```rst
.. versionadded:: 0.2
```
so we know when this function was added to the library?

Reply:
Done